sample/ios/Gemfile.lock

```diff
@@ -368,4 +368,4 @@ DEPENDENCIES
   xcov
 
 BUNDLED WITH
-   2.4.19
+   2.4.21
```

Comment:
Why do we need this change? This could lead to combability issues on iOS side.

Reply:
I think @lydiasama accidentally committed the sample folder 🤓 

FYI: the CI will automatically update all changes in the sample folder https://github.com/nimblehq/kmm-templates/blob/71cbe983c811625de8d9dae39b2a5c6653ed7266/.github/workflows/test.yml#L49-L70